src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(int radius) {
+        this.radius = radius;
+        this.setType("circle");
+    }
+
+    public double getArea() {
+        return Math.PI * radius * radius;
+    }
```

Comment:
it would be better if we had `@Override` annotation here 
(and in other classes)